package.json

```diff
@@ -1,40 +1,39 @@
 {
-  "name": "gatsby-starter-hello-world",
-  "private": true,
-  "description": "A simplified bare-bones starter for Gatsby",
-  "version": "0.1.0",
-  "license": "0BSD",
-  "scripts": {
-    "build": "gatsby build",
-    "develop": "gatsby develop",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "start": "npm run develop",
-    "serve": "gatsby serve",
-    "clean": "gatsby clean",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
-  },
-  "dependencies": {
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@mdx-js/mdx": "^1.6.16",
-    "bootstrap": "^4.5.2",
-    "gatsby": "^2.24.54",
-    "react": "^16.12.0",
-    "react-bootstrap": "^1.3.0",
-    "react-dom": "^16.12.0",
-    "react-dom-confetti": "^0.2.0",
-    "react-pose": "^4.0.10",
-    "react-router-dom": "^5.2.0",
-    "styled-components": "^5.2.0"
-  },
-  "devDependencies": {
-    "prettier": "2.1.1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/gatsbyjs/gatsby-starter-hello-world"
-  },
-  "bugs": {
-    "url": "https://github.com/gatsbyjs/gatsby/issues"
-  }
+    "name": "gatsby-starter-hello-world",
+    "private": true,
+    "description": "A simplified bare-bones starter for Gatsby",
+    "version": "0.1.0",
+    "license": "0BSD",
+    "scripts": {
+        "build": "gatsby build",
+        "develop": "gatsby develop",
+        "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+        "start": "npm run develop",
+        "serve": "gatsby serve",
+        "clean": "gatsby clean",
+        "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+    },
+    "dependencies": {
+        "@material-ui/core": "^4.11.0",
```

Comment:
Add npm install @material-ui/icons to package

Reply:
Boostrap is still in dependency list.

Reply:
I am not sure why material-ui/icons is not in the list of dependency here. I am sure master branch should have it since Demi uses icon for the footer.

Reply:
- bootstrap removed @gh0stl0nely 
- @material-ui/icons added @rishFilet